static/js/compose.js

```diff
@@ -179,6 +179,15 @@ function create_message_object() {
         message.reply_to = recipient;
         message.private_message_recipient = recipient;
         message.to_user_ids = people.email_list_to_user_ids_string(emails);
+
+        if (page_params.realm_is_zephyr_mirror_realm === false) {
```

Comment:
If you just spell this `if (!page_params...)`; we don't generally use the `=== false` pattern

Reply:
Also, I feel like we can actually do this whenever `to_user_ids` is defined; the `zephyr_mirror_realm` case is one where we only have an email address (no user ID) for our recipients